mnist/tensorflow/keras/fashion/README.md

```diff
@@ -114,21 +120,21 @@ python -m trainer.task \
     --job-dir=$JOB_DIR
 ```
 
-* **Run locally in Google Cloud ML Engine:**
+* **Google Cloud ML Engine**
 
 * **GCloud configuration:**
 
 ```
-export BUCKET_NAME=your-bucket-name
 export JOB_NAME="mnist_keras_$(date +%Y%m%d_%H%M%S)"
 export JOB_DIR=gs://$BUCKET_NAME/$JOB_NAME
-export REGION=us-central1
 export TRAIN_FILE=gs://cloud-samples-data/ml-engine/mnist/train-images-idx3-ubyte.gz
 export TRAIN_LABELS_FILE=gs://cloud-samples-data/ml-engine/mnist/train-labels-idx1-ubyte.gz
 export TEST_FILE=gs://cloud-samples-data/ml-engine/mnist/t10k-images-idx3-ubyte.gz
 export TEST_LABELS_FILE=gs://cloud-samples-data/ml-engine/mnist/t10k-labels-idx1-ubyte.gz
 ```
 
+* **Run locally in Google Cloud ML Engine:**
```

Comment:
We should rename this, it just seems confusing. 

"Run locally via the gcloud emulator for Google Cloud ML Engine"?

Reply:
ack